File: class15/python/AP1511-DiaDaSemana.py
# DiaDaSemana — Aula 15
# Retorna o nome do dia da semana (simula escolha/caso com if-elif)

dia = int(input("Digite um número (1-7): "))
if dia == 1:
    print("Domingo")
elif dia == 2:
    print("Segunda-feira")
elif dia == 3:
    print("Terça-feira")
elif dia == 4:
    print("Quarta-feira")
elif dia == 5:
    print("Quinta-feira")
elif dia == 6:
    print("Sexta-feira")
elif dia == 7:
    print("Sábado")
else:
    print("Número inválido! Digite 1-7.")

File: class15/cpp/AP1511-DiaDaSemana.cpp
#include <iostream>
int main() {
    int dia;
    std::cout << "Digite um número (1-7): "; std::cin >> dia;
    switch (dia) {
        case 1: std::cout << "Domingo" << std::endl; break;
        case 2: std::cout << "Segunda-feira" << std::endl; break;
        case 3: std::cout << "Terça-feira" << std::endl; break;
        case 4: std::cout << "Quarta-feira" << std::endl; break;
        case 5: std::cout << "Quinta-feira" << std::endl; break;
        case 6: std::cout << "Sexta-feira" << std::endl; break;
        case 7: std::cout << "Sábado" << std::endl; break;
        default: std::cout << "Número inválido! Digite 1-7." << std::endl;
    }
    return 0;
}